docs/diginstroom/aanlevering-van-sips.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+title: Digitale instroom > Aanlevering van SIPs
+has_children: true
+has_toc: true
+nav_order: 1
+has_children: true
```

Comment:
```suggestion
title: Aanlevering van SIPs
parent: Digitale instroom
has_children: false
has_toc: true
nav_order: 2
```
Aanpassingen:
- title: Enkel "aanlevering van SIPS".
- parent: Voeg de pagina toe als parent van de "Digital instroom" pagina.
- has_children: Dit stond er twee keer. Deze pagina heeft geen kinderen dus mag gerust op `false`.
- nav_order: Op twee gezet zodat die onder de "SIP spec" komt te staan in de "Digital instroom" pagina. Discussieerbaar.